app/views/foreman_ansible/job_templates/ansible_roles.erb

```diff
@@ -0,0 +1,15 @@
+<%#
+kind: job_template
+name: Ansible Roles - Ansible Default
+job_category: Ansible Playbook
+description_format: 'Run Ansible roles for host'
+feature: ansible_run_host
+provider_type: Ansible
+%>
+
+---
+- hosts: <%= @host.name %>
+  tasks:
+    - debug: msg=These are the parameters "{{ foreman_params }}"
+  roles:
+    - <%= @host.all_ansible_roles.map { |role| role.name.strip }.join("\n  - ") %>
```

Comment:
This generates the roles wrongly if `@host.all_ansible_roles` is empty. I think it should generate an empty list so applying no roles would be successful. The way it is now it fails with

```
ERROR! Invalid role definition: None
Exit status: 4
```

Reply:
I will change it, however notice that the "Run Ansible roles" button does NOT show up if a host has no Ansible roles

Reply:
That's possible, but there are other ways of triggering the run which don't have such safeguard.